charts/ark-pentaho-ce/files/setenv.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+CATALINA_OPTS="-Xms550m -Xmx1024m  -Djava.rmi.server.hostname=localhost -XX:MaxPermSize=256m -Dsun.rmi.dgc.client.gcInterval=3600000 -Dsun.rmi.dgc.server.gcInterval=3600000"

```

Comment:
Can we use `-XX:MaxRAMPercentage` instead of `-Xms` and `-Xmx`, as documented [here](https://arkcase.atlassian.net/wiki/spaces/AANTA/pages/1398046744/Best+practices+for+building+container+images#JVM-Based-Images)?

Reply:
Yeah I think we can improve this later when parameterization is implemented.